channel-azure-boards/src/test/java/com/synopsys/integration/alert/channel/azure/boards/convert/AzureBoardsGlobalConfigurationModelConverterTest.java

```diff
@@ -42,7 +42,7 @@ public void init() {
     void validConversionTest() {
         ConfigurationModel configurationModel = createDefaultConfigurationModel();
         AzureBoardsGlobalConfigurationModelConverter converter = new AzureBoardsGlobalConfigurationModelConverter(validator);
-        Optional<AzureBoardsGlobalConfigModel> model = converter.convertAndValidate(configurationModel);
+        Optional<AzureBoardsGlobalConfigModel> model = converter.convertAndValidate(configurationModel, null);
```

Comment:
We seem to be missing a test where the configuration id isn't null for an existing configuration.

Reply:
This can be added after this is merged in. It's an additional test case that didn't exist already.  Will approve for now